docker-compose.yml

```diff
@@ -5,15 +5,20 @@ services:
     image: acouvreur/ssh-log-to-influx
     build: ./
     restart: always
-    environment: 
+    environment:
       - INFLUX_URL=influxdb
       - INFLUX_DB=telegraf
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
Can you provide more details on why this variable is set ?

Reply:
This is because my influx server uses a TLS server certificate issued by a CA not known to node.js. This was simpler than figuring out how to configure TLS properly in node. This basically skips all validation of server certificates. A better solution would be to add the right CA certificate and configure the http-library accordingly, but this wasn't a priority for me...

Reply:
According to this https://node-influx.github.io/typedef/index.html#static-typedef-ISingleHostConfig the values both defaults to 'root'. So maybe it still works even if there is no variable set ?